src/objects.js

```diff
@@ -1,41 +1,59 @@
 const createPerson = (name, age) => {
   // your code here
+  this.name = name;
```

Comment:
Really good use of `this` here.

Reply:
Most people have been writing object literals like `{ name, age }`, but both work.